wandb/analytics/sentry.py

```diff
@@ -214,13 +214,12 @@ def configure_scope(
             "sweep_url",
             "sweep_id",
             "deployment",
-            "_disable_service",
-            "_require_legacy_service",
+            "x_disable_service",
+            "x_require_legacy_service",
             "launch",
+            "_platform",
```

Comment:
```suggestion
            "x_platform",
```
should be?

Reply:
oh is it computed?

Reply:
no, it's computed. we may prefix them all with `c_` or something, but in a next pr.